.circleci/config.yml

```diff
@@ -35,6 +34,10 @@ jobs:
       - terraform/fmt:
           path: "."
           #checkout: true
+  validate-terraform-install:
+    machine: true
```

Comment:
Can we use this machine image: ubuntu-2004:202010-01
True unfortunately defaults to Ubuntu 14.04
https://circleci.com/docs/2.0/configuration-reference/#available-machine-images